### PR TITLE
Add Citus integration smoke test

### PR DESCRIPTION
**Pull request by msakrejda**
This adds a very basic run-through of pganalyze on a Citus
workload. It uses the latest Citus image and modifies the default
pgbench schema we use in stock Postgres tests to use Citus distributed
tables. This ensures monitoring features like calculating distributed
table sizes on Citus are properly exercised.

### CODE REVIEW

**Comment by msakrejda** [comment]
This is blocked by #335; will rebase once that lands.

**Review by lfittl — APPROVED** [review]
:+1:

**Pull request merged**
